tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java

```diff
@@ -56,6 +75,9 @@ public class LambdaToS3StepDefinitions {
 
     JSONObject enrichedJSON;
 
+    static String sqsoutputtext;
+    static String SqsUrl ="https://sqs.eu-west-2.amazonaws.com/750703655225/PublishToAccountsSQSQueue-build";
```

Comment:
we should not be hard coding urls as it is sensitive information. check slack for guidance

Reply:
The URL is now added to runtests.sh file with a separate configuration reader file.